wrc/codegen/cghtml.py

```diff
@@ -89,11 +89,11 @@ def simple_md2html(text, urls):
 class WCADocumentHtml(CGDocument):
     ''' Emit html formatted to fit in the WCA website.  '''
     name = "HTML"
+
     def __init__(self, versionhash, language, pdf):
         super(WCADocumentHtml, self).__init__(str)
         self.regset = set()
-        self.urls = {'regulations': './', 'guidelines': './guidelines.html',
-                     'pdf': pdf}
+        self.urls = {'regulations': './', 'guidelines': './guidelines.html', 'pdf': pdf}
```

Comment:
If we are going to implement this, we'll need to change this line too.

Reply:
I first need to know the URL of the new static page (i.e. './merged.html' or whatever).

Reply:
We will discuss this and I will ping you here once I have the answer.